src/components/Paginate.tsx

```diff
@@ -0,0 +1,213 @@
+import { chevronBackSharp, chevronForwardSharp } from "ionicons/icons";
+import { IonIcon } from "@ionic/react";
+import React, { FunctionComponent, useEffect, useState } from "react";
+
+const getPageCount = (
+  numberOfResults: number,
+  numberOfResultsPerPage: number
+): number => Math.ceil(numberOfResults / numberOfResultsPerPage);
```

Comment:
Can we shorten `numberOfResultsPerPage`? Also fn name uses "count" and param name uses "number", which I think is slightly inconsistent. Any ideas for shorter and more consistent names?

Reply:
> Can we shorten `numberOfResultsPerPage`?

We can change `numberOfResultsPerPage` to `resultsPerPage`. af17000d5f5f52112a16d0afcfe7a889a62df4c9 does this.

> Also fn name uses "count" and param name uses "number", which I think is slightly inconsistent. Any ideas for shorter and more consistent names?

We can call `getPageCount` `getNumPages`, instead, and call `numberOfResults` `numResults`, instead. 83335a31b644303d4944fc998dafa54a7bba5fca does this.

Reply:
😍 much more readable. This also follows principle #4 well (Use shorthands when possible).

Reply:
Thank you for the recommendations!